### ISSUE TITLE
Regression: Unconditionally split `hasOwnProperty` call to multiple lines.

### ISSUE THREAD

**Issue by IvanGoncharov** [prettier/prettier#4602]
**Prettier 1.13.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAEALAhmgNVwBsBXOTAXh3wHkB3KABQCcIAHOFmATwApcAGkwAjAJQBuEIJAcYAS3TJQuFm3pNVCNMhC4AbhHkATaSBEtcYANZwYAZXZX5UAObIYLcjJdouMVlxXAFtcZAAzEj8ZACs0AA8AIUsbO3tcYLgAGRc4CKi4GScWPxZdEVwRHmJoM3YWFxgAdRMYbGQADgAGIrY-Jst2XXq4Uv08mRY4AEdSeSnAkLCkSOJokD9g+Q8vQo2XV2I4AEVSCHh8tb2YSpbjNuQAJhlPXHliA4BhCGDQ3ShoBMQKQ-AAVSo6FYFAC+0KAA)
```sh
--parser babylon
```

**Input:**
```jsx
const hasValue = hasOwnProperty(a, b);
```

**Output:**
```jsx
const hasValue = hasOwnProperty(
  a,
  b
);

```

**Expected behavior:**
```jsx
const hasValue = hasOwnProperty(a, b);
```

**Comment by IvanGoncharov** [comment]
Regression happened when I updated from `0.12.1` to `1.13.2`.
The same behavior occurs for `toString`, `propertyIsEnumerable`, etc.

**Comment by duailibe** [comment]
oops sorry @ikatyang go ahead

**Comment by IvanGoncharov** [comment]
I suspect you are using `{}` instead of `Object.create(null)` to index represent a map with function name as a key.

**Comment by duailibe** [comment]
This is fixed in 1.13.3.
